README.md

```diff
@@ -30,7 +30,70 @@ Until [WeakReference](https://ponyfoo.com/articles/weakref) is added to the Java
 
 ## Usage Example
 
-Currently our best examples are in the test folder.
+We have an example in [the example folder](./src/example).
+
+It works like this:
+
+```typescript
+// server.js
```

Comment:
Curious why this is JS instead of TS?  Also, I think this is the client file and the other one is the server file?  Looking at the code, this one connects to the other and fetches its API, which implies to me this is the client.